reporters/GlobalSummaryReporter.js

```diff
@@ -0,0 +1,237 @@
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.collateralAmount));
+      }
+    }
+    for (let event of expirySettlementEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.collateralReturned));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.collateralReturned));
+      }
+    }
+    for (let event of liquidationRewardEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.withdrawalAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.withdrawalAmount));
+      }
+    }
+    for (let event of regularFeeEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.regularFee)).add(toBN(event.lateFee));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.regularFee)).add(toBN(event.lateFee));
+      }
+    }
+    for (let event of finalFeeEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.amount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.amount));
+      }
+    }
+
+    allSponsorStatsTable["collateral withdrawn"] = {
+      cumulative: this.formatDecimalString(collateralWithdrawn),
+      "24H": this.formatDecimalString(collateralWithdrawnDaily)
+    };
+
+    // - Net collateral deposited into contract:
+    let netCollateralWithdrawn = collateralDeposited.sub(collateralWithdrawn);
+    let netCollateralWithdrawnDaily = collateralDepositedDaily.sub(collateralWithdrawnDaily);
+    allSponsorStatsTable["net collateral deposited"] = {
+      cumulative: this.formatDecimalString(netCollateralWithdrawn),
+      "24H": this.formatDecimalString(netCollateralWithdrawnDaily)
+    };
+
+    // - Tokens minted: Creates
+    let tokensMinted = toBN("0");
+    let tokensMintedDaily = toBN("0");
+    for (let event of createEvents) {
+      tokensMinted = tokensMinted.add(toBN(event.tokenAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        tokensMintedDaily = tokensMintedDaily.add(toBN(event.collateralAmount));
+      }
+    }
+    allSponsorStatsTable["tokens minted"] = {
+      cumulative: this.formatDecimalString(tokensMinted),
+      "24H": this.formatDecimalString(tokensMintedDaily),
+      current: this.formatDecimalString(await this.empContract.methods.totalTokensOutstanding().call())
+    };
+
+    // - Tokens repaid: Redeems, SettleExpired's
+    let tokensRepaid = toBN("0");
+    let tokensRepaidDaily = toBN("0");
+    for (let event of redeemEvents) {
+      tokensRepaid = tokensRepaid.add(toBN(event.tokenAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        tokensRepaidDaily = tokensRepaidDaily.add(toBN(event.collateralAmount));
```

Comment:
nit: same as above -- shouldn't this be `tokenAmount`?

Reply:
yep